docs/docs/config/battery.md

```diff
@@ -49,22 +49,42 @@ Applies to: [`/chosen` node](https://docs.zephyrproject.org/3.5.0/build/dts/intr
 
 ## Battery Voltage Divider Sensor
 
-Driver for reading the voltage of a battery using an ADC connected to a voltage divider.
+Driver for reading the voltage of a battery using an ADC connected to a voltage divider. This driver can also read a GPIO pin to detect whether the battery is charging or not. This requires supported hardware (a battery charging IC with an output to indicate charging status). This functionality is optional, if the hardware doesn't support it the `chg-gpios` devicetree configuration does not have to be set.
 
 ### Devicetree
 
 Applies to: `compatible = "zmk,battery-voltage-divider"`
 
-See [Zephyr's voltage divider documentation](https://docs.zephyrproject.org/3.5.0/build/dts/api/bindings/iio/afe/voltage-divider.html).
+Definition file: [zmk/app/module/dts/bindings/sensor/zmk,battery-voltage-divider.yaml](https://github.com/zmkfirmware/zmk/blob/main/app/module/dts/bindings/sensor/zmk,battery-voltage-divider.yaml)
+
+The ZMK battery voltage divider includes the [Zephyr voltage divider](https://docs.zephyrproject.org/latest/build/dts/api/bindings/adc/voltage-divider.html) and adds on additional functionality.
+
+| Property    | Type       | Description                                      | Default |
+| ----------- | ---------- | ------------------------------------------------ | ------- |
+| `chg-gpios` | GPIO array | GPIO connected to the charging IC's charging pin |         |
+
+:::note Charging indication
+
+The battery charging status is not consumed by any built-in indicators currently and it cannot be conveyed to the host over BLE.
```

Comment:
I wonder if this is a bit confusing, because the widgets still show a charging indicator tied to the USB status, right?

Reply:
I considered converting all the display widgets in this pr but that's already been done in #2153 and i didn't want to document a feature that didn't exist yet or hijack someone else's PR. If #2153 gets merged first I'll rebase this with that in mind, if this gets merged first the other PR can document it

Reply:
Makes sense, let's see how it goes.

Reply:
yea i think whichever one gets merged first, the other one will just rebase on top